### PR TITLE
fix: broken link in pNFT guide

### PR DESCRIPTION
**Pull request by jim4067**
Update broken links, to point to this ☝️ repository [Programmable NFT Guide](https://github.com/metaplex-foundation/mpl-token-metadata/blob/main/programs/token-metadata/program/ProgrammableNFTGuide.md)

### CODE REVIEW

**Review by lorisleiva — DISMISSED** [review]
Thank you!

**Review by danenbm — APPROVED** [review]

File: programs/token-metadata/program/ProgrammableNFTGuide.md [hunk 1]
```diff
@@ -13,22 +13,22 @@ Token Authorization Rules
...
+* [x] [`Lock`](https://github.com/metaplex-foundation/mpl-token-metadata/blob/45826f100c82bc2333116fed53f9a820a5329293/programs/token-metadata/program/src/instruction/mod.rs#L642-L660) <span style="font-family:'Lucida Console', monospace; font-size: 6pt">([TypeScript](https://github.com/metaplex-foundation/mpl-token-metadata/blob/main/programs/token-metadata/js/test/lock.test.ts) | [Rust](https://github.com/metaplex-foundation/mpl-token-metadata/blob/main/programs/token-metadata/program/tests/lock.rs))</span>
+* [x] [`Migrate`](https://github.com/metaplex-foundation/mpl-token-metadata/blob/45826f100c82bc2333116fed53f9a820a5329293/programs/token-metadata/program/src/instruction/mod.rs#L682-L6993)
```
Comment (febo) [inline programs/token-metadata/program/ProgrammableNFTGuide.md:22]: The line offset does not seem correct.

File: programs/token-metadata/program/ProgrammableNFTGuide.md [hunk 2]
```diff
@@ -13,22 +13,22 @@ Token Authorization Rules
...
-* [x] [`Unverify`](https://github.com/metaplex-foundation/metaplex-program-library/blob/master/token-metadata/program/src/instruction/mod.rs#L773-L785) <span style="font-family:'Lucida Console', monospace; font-size: 6pt">([TypeScript](https://github.com/metaplex-foundation/metaplex-program-library/blob/master/token-metadata/js/test/verification.test.ts) | [Rust](https://github.com/metaplex-foundation/metaplex-program-library/blob/master/token-metadata/program/tests/unverify.rs))</span>
+* [x] [`Mint`](https://github.com/metaplex-foundation/mpl-token-metadata/blob/45826f100c82bc2333116fed53f9a820a5329293/programs/token-metadata/program/src/instruction/mod.rs#L573-L595) <span style="font-family:'Lucida Console', monospace; font-size: 6pt">([TypeScript](https://github.com/metaplex-foundation/mpl-token-metadata/blob/main/programs/token-metadata/js/test/mint.test.ts) | [Rust](https://github.com/metaplex-foundation/mpl-token-metadata/blob/main/programs/token-metadata/program/tests/mint.rs))</span>
```
Comment (febo) [inline programs/token-metadata/program/ProgrammableNFTGuide.md:24]: Missing the link for the `Print` instruction.

**Review by febo — CHANGES_REQUESTED** [review]

**Review by febo — APPROVED** [review]
Thank you!

**Review by lorisleiva — APPROVED** [review]

**Pull request merged**